### PR TITLE
ADAPT-3254 | @mattanglin | basic trip entity

### PR DESCRIPTION
**Pull request by mattanglin**
# READY FOR REVIEW

# Summary
- Save Storyblok component schema to repository
- Adds Basic Trip entity
-  https://stanfordits.atlassian.net/browse/ADAPT-3254

# Review By (Date)
- Asap

# Review Tasks

## Setup tasks and/or behavior to test

1. View the Storyblok Trip component in the [DEV SAA Alumni space](https://app.storyblok.com/#!/me/spaces/118608/components)

**NOTE: This component is not final and will likely evolve as additional tickets are completed. This ticket simply establishes a starting point as well as enforcing the tracking of components as code.**

### CODE REVIEW

File: storyblok/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+```
+> npm run sb:pull
+```
+
+This will update the Storyblok component schema and preset files stored in the `storyblok/` directory.
+
+## Push Components
+
+We have not currently we have not integrated pushing components back to the production space at this time.
```
Comment (yvonnetangsu) [inline storyblok/README.md:25]:
```suggestion
We have not integrated pushing components back to the production space at this time.
```

File: storyblok/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Storyblok Components
+
+This directory contains our Storyblok component schemas as pulled from the dev space.
+If you update a Storyblok component during your work please update the schema file.
+
+## Pull COmponents
```
Comment (yvonnetangsu) [inline storyblok/README.md:6]:
```suggestion
## Pull Components
```

**Review by yvonnetangsu — APPROVED** [review]
Looks good to me, @mattanglin ! Just a couple typos but preapproving - please feel free to merge.

**Comment by sherakama** [comment]
Ooo, I am excited to see the component code tracking. Looking forward to how this keeps getting better.

**Comment by yvonnetangsu** [comment]
I'm testing the Storyblok CLI in my collection card branch, but getting the same error that this person is getting:
https://github.com/storyblok/storyblok/issues/691

I've set my space ID, tried both the new `npm run sb:pull` script, as well as using the full command, but getting a 401 error 🤔  I don't think there's anything wrong with your new script, @mattanglin, but I have't figured out what's happening yet. I've used the Storyblok CLI before and it worked back then (long time ago).
Screen Shot 2021-09-07 at 11 25 32 PM

**Comment by sherakama** [comment]
Try logging out and in again and then try to list spaces.
```
$ storyblok logout
$ storyblok login
$ storyblok spaces
```

**Comment by sherakama** [comment]
401 is an unauthorized response so it is likely due to your user

**Comment by sherakama** [comment]
Also, you cannot log in with an SSO account. You will need a regular account.

**Comment by yvonnetangsu** [comment]
> Also, you cannot log in with an SSO account. You will need a regular account.

😬  That's probably it. Thanks!

**Comment by sherakama** [comment]
https://github.com/storyblok/storyblok/issues/698

**Comment by yvonnetangsu** [comment]
Oh you already open the issue a month ago @sherakama 😄 ! I hope they do something about it.

**Comment by sherakama** [comment]
You can also leave them some feedback here: https://feedback.storyblok.com/b/feature-requests/

**Comment by yvonnetangsu** [comment]
> You can also leave them some feedback here: https://feedback.storyblok.com/b/feature-requests/

Sure. Will do!

**Pull request merged**